pulsar-common/pom.xml

```diff
@@ -196,6 +196,24 @@
 
   <build>
   <plugins>
+    <plugin>
+      <groupId>org.gaul</groupId>
+      <artifactId>modernizer-maven-plugin</artifactId>
```

Comment:
We should define the plugin in the parent pom

Reply:
If it's on parent pom, it will mean all the changes have to be done in a single commit which is not good. 

Though the plugin version should be defined in the pluginManagement section in parent pom

Reply:
Yes I meant to use pluginManagement  in order to set the version.